linux_os/guide/services/ntp/service_chronyd_enabled/rule.yml

```diff
@@ -29,6 +29,7 @@ references:
     cis@rhel7: 2.2.1.3
     cis@rhel8: 2.2.1.2
     ism: 0988,1405
+    srg: SRG-OS-000480-GPOS-00227
```

Comment:
I have found the `service_chronyd_enabled` rule in the `SRG-OS-000355-GPOS-00143` instead of `SRG-OS-000480-GPOS-00227`. Seems the `SRG-OS-000355-GPOS-00143` would be the correct one, right?

Reply:
You are correct, fixed.